.github/workflows/update-version.yml

```diff
@@ -1,48 +1,56 @@
-#
-# Copyright 2024 Adobe. All rights reserved.
-# This file is licensed to you under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License. You may obtain a copy
-# of the License at http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software distributed under
-# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
-# OF ANY KIND, either express or implied. See the License for the specific language
-# governing permissions and limitations under the License.
-#
+name: Update Versions
 
-name: Update Version
 on:
   workflow_dispatch:
     inputs:
-      version: 
-        description: 'New version to use for the Edge extension. Example: 3.0.0'
+      version:
+        description: 'New version to use for the extension (ex: 1.2.3).'
+        type: string
         required: true
+        default: ''
+
+      branch:
+        description: 'The target branch where the version update will be applied and the pull request will be merged into.'
+        type: string
+        required: true
+        default: ''
 
       core-dependency:
-        description: '[Optional] Update Core dependency in pom.xml. Example: 3.0.0'
+        description: 'If a version is provided, update AEPCore dependency in gradle.properties (ex: 1.2.3).'
+        type: string
         required: false
+        default: ''
 
-      identity-dependency:
-        description: '[Optional] Update Edge Identity dependency in pom.xml. Example: 3.0.0'
+      edge-identity-dependency:
```

Comment:
Specify that this is a dependency in the pom.xm file, so users know this is a publish dependency.